build/template.tex

```diff
@@ -76,29 +94,59 @@
 
 % LITERATURVERZEICHNIS
 % TODO Formatierung
+
+% Im Kusche-Mode wird arabische Nummerierung beibehalten
+\if\CKUSCHE 0
 \pagenumbering	{Roman} \setcounter{page}{\thesavepage}
+\fi
 
-% wenn keine literatur verwendet (zitiert) wird, erstelle kein literaturverzeichnis 
-\ifnum\thetotalbibentries>0
-    \printbibliography[title=Literaturverzeichnis]
-    \addcontentsline{toc}{section}{Literaturverzeichnis}
-    \cleardoublepage
+% wenn keine Literatur verwendet (zitiert) wird, erstelle kein Literaturverzeichnis
+% im Kusche-Mode wird das LiteraturVZ später generiert
+\if\CKUSCHE 0
+    \ifnum\thetotalbibentries>0
+        \printbibliography[title=Literaturverzeichnis]
+        \addcontentsline{toc}{section}{Literaturverzeichnis}
+        \cleardoublepage
+    \fi
 \fi
 
 
 % TODO %ANLAGENVERZEICHNIS UND ANLAGEN
-\ifnum\totvalue{anlagenentries}>0
-    \section*{Anlagen}
-    \begin{spacing}{2}
-        \listofanlagen	
-    \end{spacing}
-    \addcontentsline{toc}{section}{Anlagen}
-    \cleardoublepage
+% erzeugt ein Anlagenkapitel und fügt es zum InhaltsVZ hinzu, insofern Anlagen existieren.
+
+% \listofanlagen generiert ein Anlagenverzeichnis
+% im Kusche-Modus wird kein Anlagenverzeichnis erzeugt, aber Anlagen im Inhaltsverzeichnis geführt (TODO)
+    \ifnum\totvalue{anlagenentries}>0
+        \section*{Anlagen}
+        \if\CKUSCHE 0
+            \begin{spacing}{2}
+                \listofanlagen
+            \end{spacing}
+        \fi
+        \addcontentsline{toc}{section}{Anlagen}
+        \cleardoublepage
+\fi
+
+\if\CKUSCHE 1
+    % die kapitel.lfd-Nummerierung wird im Kusche Mode ab hier wieder deaktiviert,
+    % da Anhänge mit Buchstaben nummeriert werden sollen
+    \counterwithout{figure}{section}
+    \counterwithout{table}{section}
 \fi
-\renewFigTabCap % change the behavior from the env figure and table and the command \caption{}
+
+\renewFigTabCap % Verhalten von Figure und Table Environments sowie \caption wird verändert (TODO: wie genau? LG, ZPM)

```

Comment:
Uff, gute Frage ... schau ich mir in Ruhe nochmal an. Aber guter Gedanke, die Kommentar mal auszubessern.

Reply:
Das sollte das Verhalten der Figure und Table nur ab dem Anlageverzeichnissteil verändert, damit diese dort einfach verwendet werden können und dann automatisch im Verzeichniss erscheinen :D (Das ist auch der Grund warum danach direkt die `anlagen.tex` importiert wird)

Reply:
Achso, also habe ich das richtig so verstanden: der Befehl sorgt dafür, dass ein neuer Zähler für Anlagen verwendet wird, die ja zusammen gezählt werden, und die Caption wird angepasst, weil der AddToAnlagen "ATA" Befehl das so braucht?

Reply:
Letztendlich muss der `ATA` Befehl nur aufgerufen werden um einen Eintrag im neuem erzeugten TOC hinzuzufügen. 
Und da `captions` bei einer `figure` oder `table` env required sind ist er dort untergebracht^^.

Reply:
Wird separat gelöst (https://github.com/RvNovae/dhge-latex/tree/fpr-qol-updates)